### DISCUSSION TITLE
cPanel Nodejs app installer for Faustjs

### DISCUSSION THREAD

**Discussion by hamiednasser** [wpengine/faustjs#1747] [General]
How to install Faustjs using cPanel Nodejs app installer.
What is the Application startup file for Faustjs?

**Comment by TeresaGobble** [comment]
Hello @hamiednasser , at this time we do not support use of cPanel Nodejs with Faustjs, but I have noted this for team discussion and appreciate your reaching out. You may find more context to explore how others use Faustjs with various stacks by visiting the Discord community: https://discord.com/invite/J2khkF9XYK

**Comment by alecarg** [comment]
+1
